src/current/cockroachcloud/cloud-api.md

```diff
@@ -152,138 +167,267 @@ curl --request POST \
+For example, to create a new Standard cluster named `notorious-moose` using the default values for the cloud provider and region:
 
-To retrieve detailed information about a specific cluster, make a `GET` request to the `/v1/clusters/{cluster_id}` endpoint.
+<div class="filters clearfix">
+    <button class="filter-button page-level" data-scope="curl"><strong>curl</strong></button>
+    <button class="filter-button page-level" data-scope="raw"><strong>Raw</strong></button>
+</div>
+
+<section class="filter-content" markdown="1" data-scope="curl">
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+curl --request POST \
+  --url https://cockroachlabs.cloud/api/v1/clusters \
+  --header 'Authorization: Bearer {secret_key}' \
+  --json '{"name":"notorious-moose","provider":"GCP","plan":"STANDARD","spec":{"serverless":{"regions":["us-central1"],"usage_limits":{"provisioned_virtual_cpus":"2"}}}}'
+~~~
+
+</section>
+
+<section class="filter-content" markdown="1" data-scope="raw">
+
+{% include_cached copy-clipboard.html %}
+~~~ JSON
+{
+  "name": "notorious-moose",
+  "provider": "GCP",
+  "plan": "STANDARD",
+  "spec": {
+    "serverless": {
+      "regions": [
+        "us-central1"
+      ],
+      "usage_limits": {
+        "provisioned_virtual_cpus": "2"
+      }
+    }
+  }
+}
+~~~
+
+</section>
+
+If the request was successful, the API returns information about the new cluster.
+
+For details about returned fields, refer to the [response example and schema](https://www.cockroachlabs.com/docs/api/cloud/v1.html#post-/api/v1/clusters) in the API reference.
+
+### Create an Advanced cluster
+
+To create a cluster, send a `POST` request to the `/v1/clusters` endpoint.
 
 {{site.data.alerts.callout_success}}
-The service account associated with the secret key must have the Cluster Administrator or Cluster Developer [role]({% link cockroachcloud/authorization.md %}#organization-user-roles).
+The service account associated with the secret key must have the Cluster Administrator or Cluster Creator [role]({% link cockroachcloud/authorization.md %}#organization-user-roles).
 {{site.data.alerts.end}}
 
+<div class="filters clearfix">
+    <button class="filter-button page-level" data-scope="curl"><strong>curl</strong></button>
+    <button class="filter-button page-level" data-scope="raw"><strong>Raw</strong></button>
+</div>
+
+<section class="filter-content" markdown="1" data-scope="curl">
+
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-curl --request GET \
-  --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id} \
-  --header 'Authorization: Bearer {secret_key}'
+curl --request POST \
+  --url https://cockroachlabs.cloud/api/v1/clusters \
+  --header 'Authorization: Bearer {secret_key}' \
+  --json '{"name":"{cluster_name}","provider":"{cloud_provider}","plan":"ADVANCED","spec":{"dedicated":{"region_nodes":{"{region_name}":3},"hardware":{"machine_spec":{"machine_type":"{machine_type}"}},"cockroach_version":"{version}"}}}'
```

Comment:
We should not be encouraging users to use `machine_spec` with Advanced. Instead, we should be pointing them towards `num_virtual_cpus`.

Reply:
Done